config/bolt/contenttypes.yaml

```diff
@@ -204,6 +204,34 @@ blockquotes:
     icon_many: "fa:quote-left"
     icon_one: "fa:quote-left"
     viewless: true
+
+showcases:
+    name: Showcases
+    singular_name: Showcase
+    fields:
+        title:
+            type: text
+            class: large
+        slug:
+            type: slug
+            uses: title
+        website_name:
+            type: text
+        website_link:
+            type: text
+            label: Link
+            placeholder: 'http://example.org/'
+        screenshot:
+            type: image
+        publication_date:
+            type: date
+            mode: datetime
+        contributor:

```

Comment:
That's great! Something I failed to mention is that, for some people, we want to link to the people.boltcms.io website, and for some not.

Can we make this field `redactor`, so that the name can either be a simple text (like now) or text with a link?

Reply:
done

Reply:
Do we want to sort on order or another field? With the `-`?

Reply:
Looks good, thanks!